kubeflow/jupyter/prototypes/jupyter-web-app.jsonnet

```diff
@@ -8,6 +8,8 @@
 // @optionalParam policy string Always imagePullPolicy for the UI's image
 // @optionalParam prefix string jupyter The prefix under which the app is accessed
 // @optionalParam rokSecretName string secret-rok-{username} The name of the secret containing user's credentials for Arrikto Rok
+// @optionalParam injectIstio string false Whether to inject istio sidecar; should be true or false.
+// @optionalParam clusterDomain string svc.cluster.local DNS config to cluster domain.
```

Comment:
do we have to do this clusterDomain for all components?

Reply:
yap, and I did add it to all components.